tests/api/items/get_by_bbox.test.ts

```diff
@@ -0,0 +1,31 @@
+import 'should'
+import { map } from 'lodash-es'
+import { createItem } from '#fixtures/items'
+import { createUser, getRandomPosition } from '#fixtures/users'
+import { fixedEncodeURIComponent } from '#lib/utils/url'
+import { waitForIndexation } from '#tests/api/utils/search'
+import { publicReq } from '#tests/api/utils/utils'
+import type { LatLng } from '#types/common'
+
+const endpoint = '/api/items?action=by-bbox'
+
+const userPosition: LatLng = getRandomPosition()
+const [ lat, lng ] = userPosition
+const geolocatedUserPromise = createUser({ position: userPosition })
+const bbox: string = fixedEncodeURIComponent(JSON.stringify([
+  lng - 1, // minLng
+  lat - 1, // minLat
+  lng + 1, // maxLng
+  lat + 1, // maxLat
+]))
+
+describe('items:public:by-bbox', () => {
+  it('should get anonymized public items nearby', async () => {
```

Comment:
what's "anonymized" about those items?

Reply:
Legacy from former implementation, removed in 32114a27e